x/stake/store.md

```diff
@@ -0,0 +1,33 @@
+# Stores
+
+This document provided a bit more insight as to the purpose of several related
+prefixed areas of the staking store which are accessed in `x/stake/keeper.go`.
+
+
+## Validators 
+ - Prefix Key Space:    ValidatorsKey
+ - Key/Sort:            Validator Owner Address
+ - Contains:            All Validator records whether independent of being bonded or not
```

Comment:
"whether bonded or not" or "independent of bonding status"

Reply:
independent of bonding status. thx